orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/utils/HealthHelper.groovy

```diff
@@ -82,6 +82,21 @@ class HealthHelper {
     } as Map
   }
 
+  static boolean someAreDown(Map instance, Collection<String> interestingHealthProviderNames) {
+    List<Map> healths = filterHealths(instance, interestingHealthProviderNames)
+
+    if (!interestingHealthProviderNames && !healths) {
```

Comment:
Good catch, thank you. 

I moved this method into the `WaitForUpInstancesTask`, since it's only used there and only makes sense when checking for instances to be in an up state. I also added logic upstream of the call so, if `interestingHealthProviderNames` is an empty list, we declare all instances to be up